samples/CMakeLists.txt

```diff
@@ -38,6 +39,7 @@ install(DIRECTORY
             python/greedy_causal_lm
             python/multinomial_causal_lm
             python/whisper_speech_recognition
+            cpp/speculative_decoding_lm
```

Comment:
this is only for python samples. You have installed `cpp/speculative_decoding_lm` above.